### ISSUE TITLE
Azure function app zip

### ISSUE THREAD

**Issue by arichiardi** [juxt/pack.alpha#18]
Hi!

I am looking for a reliable way to package my function app and ended up looking at what you already have in here.
I will state the problem and would like to ask you suggestions on how to eventually implement it.

Azure function apps are a bit different from AWS, the structure being:

```
- package.zip
  - host.json
  - cljs        (dir containing the runtime, shared)
  - node_modules (this can go anywhere)
  - lamdba-foo
    - function.json
    - index.js
  - lambda-bar
    - function.json
    - index.js
```

As you can see there are a couple of quirks. I am using shadow for compilation and it also allows you to specify a build her where the above will end up folder - you could have `node_modules` in the root of the project and the functions in `my-build-dir`.

I hope this is clear enough, if not I can answer all sorts of questions and commit to implementing the packager.

Thank you Dominic/Just team!

**Comment by SevereOverfl0w** [comment]
It appears in this case that there's no dependencies from maven, is that correct?

**Comment by arichiardi** [comment]
Correct, I might be thought you do the same for AWS, but I realize now that you are talking about JVM lambdas and I was thinking JS lambdas.

**Comment by SevereOverfl0w** [comment]
My gut feeling is that using the `zip` command is going to be much easier than using pack for this.

**Comment by arichiardi** [comment]
Yeah well, I am actually inquiring here because of many of the zip limitations like no control on zipped paths - have to cd in and our folders... probably I will need to use a small lumo wrapper or something...
